garrysmod/gamemodes/sandbox/gamemode/spawnmenu/controlpanel.lua

```diff
@@ -91,6 +91,27 @@ function PANEL:KeyBinder( label1, convar1, label2, convar2 )
 
 end
 
+function PANEL:ColorBox(label, convarr, convarg, convarb, convara)

```

Comment:
Coding style: Add spaces: `( label, ... , convara )`

Also possibly rename to `ColorPicker`?

Reply:
Sure thing